runbotics/runbotics-orchestrator-ui/src/src-app/views/credentials/Credential/EditCredential/CredentialAttribute/Attribute.styles.ts

```diff
@@ -0,0 +1,51 @@
+import { Grid, Card, Typography, IconButton, CardContent } from '@mui/material';
+import { grey } from '@mui/material/colors';
+import styled from 'styled-components';
+
+export const StyledGridContainer = styled(Grid)(
+    ({ theme }) => `
+    padding: ${theme.spacing(1)};
+    `
+);
+
+export const StyledAttributeCard = styled(Card)(
+    ({ theme }) => `
+    background-color: ${grey[100]};
+    border: 1px solid ${grey[400]};
+    padding: ${theme.spacing(1)};
+`
+);
+
+export const StyledAttributeCardContent = styled(CardContent)`
+    flex-grow: 1
+`;
+
+export const AttributeInfoNotEdiable = styled(Typography)`
+    &.MuiTypography-root {
+        margin-left: 0.5rem;
```

Comment:
this is true - I shuld use theme.spacing here, but since it is not used anymore I removed this styled component entirely (was used when I was doing custom and template attributes - forgot to remove)